src/main/cpp/chassis/driveStates/TrajectoryDrivePathPlanner.cpp

```diff
@@ -53,6 +53,7 @@ TrajectoryDrivePathPlanner::TrajectoryDrivePathPlanner(RobotDrive *robotDrive) :
 
 void TrajectoryDrivePathPlanner::Init(ChassisMovement &chassisMovement)
 {
+    m_firstGen++;
```

Comment:
Why incrementing?   If we have 3 paths in an auton, this gets incremented each time we encounter a new path.

Reply:
We can talk about this one